test/xdr/ledger_entries/account_entry_ext_test.exs

```diff
@@ -132,12 +135,21 @@ defmodule StellarBase.XDR.AccountEntryExtTest do
     end
   end
 
-  @spec create_account_id_list() :: AccountIDList.t()
-  defp create_account_id_list do
-    account_id_1 = create_account_id("GCNY5OXYSY4FKHOPT2SPOQZAOEIGXB5LBYW3HVU3OWSTQITS65M5RCNY")
-    account_id_2 = create_account_id("GBZNLMUQMIN3VGUJISKZU7GNY3O3XLMYEHJCKCSMDHKLGSMKALRXOEZD")
-
-    [account_id_1, account_id_2]
-    |> AccountIDList.new()
+  @spec create_sponsorship_descriptor_list() :: SponsorshipDescriptorList.t()
+  defp create_sponsorship_descriptor_list do
+    sponsorship_descriptor_1 =
+      "GCNY5OXYSY4FKHOPT2SPOQZAOEIGXB5LBYW3HVU3OWSTQITS65M5RCNY"
+      |> create_account_id()
+      |> OptionalAccountID.new()
+      |> SponsorshipDescriptor.new()
+
+    sponsorship_descriptor_2 =
+      "GBZNLMUQMIN3VGUJISKZU7GNY3O3XLMYEHJCKCSMDHKLGSMKALRXOEZD"
+      |> create_account_id()
+      |> OptionalAccountID.new()
+      |> SponsorshipDescriptor.new()
+
+    [sponsorship_descriptor_1, sponsorship_descriptor_2]
+    |> SponsorshipDescriptorList.new()
```

Comment:
A pipeline here may not be necessary, what if we just call the function like this?
```elixir
SponsorshipDescriptorList.new([sponsorship_descriptor_1, sponsorship_descriptor_2])
```